Tests/Functional/app/Configuration/config.yml

```diff
@@ -32,6 +32,7 @@ security:
     firewalls:
         default:
             anonymous: ~
+            logout_on_user_change: true
```

Comment:
this will likely fail with pre 3.4 Symfony versions

Reply:
I guess there is no easy fix for this one.

Reply:
I'm gonna take a look into it.